### PR TITLE
Shore up dbt docs

### PR DESCRIPTION
**Pull request by dataders**
## overview of changes
1. lowercase styliation, `dbt`, is preferred over `DBT`
2. include boilerplate overview of dbt
3. ~~`dbt-synapsevnext`~~ -> `dbt-fabric`

### CODE REVIEW

File: docs/data-warehouse/connectivity.md [hunk 1]
```diff
@@ -118,13 +118,15 @@ When establishing connectivity via JDBC, check for the following dependencies:
     </dependency>
     ```
 
-## Connect using DBT
+## Connect using dbt
 
-Users typically use DBT adapters to connect DBT projects to a target datastore. DBT adapters are built specifically for each data source. Users who would like to connect to Synapse Data Warehouse in Microsoft [!INCLUDE [product-name](../includes/product-name.md)] from DBT project must use the `dbt-synapsevnext` DBT adapter. Similarly, the Azure Synapse Analytics dedicated SQL pool data source has its own adapter, `dbt-synapse`.
+dbt is a data transformation framework that uses software engineering best practices like testing and version control to reduce code, automate dependency management, and ship more reliable data—all with SQL.
```
Comment (WilliamDAssafMSFT) [inline docs/data-warehouse/connectivity.md:123]:
```suggestion
The `dbt` adapter is a data transformation framework that uses software engineering best practices like testing and version control to reduce code, automate dependency management, and ship more reliable data—all with SQL.
```

File: docs/data-warehouse/connectivity.md [hunk 2]
```diff
@@ -118,13 +118,15 @@ When establishing connectivity via JDBC, check for the following dependencies:
...
 
-The DBT Fabric DW Adapter uses the `pyodbc` library to establish connectivity with the [!INCLUDE [fabric-dw](includes/fabric-dw.md)]. The `pyodbc` library is an ODBC implementation in Python language that uses [Python Database API Specification v2.0](https://peps.python.org/pep-0249/).  The `pyodbc` library directly passes connection string to the database driver through SQLDriverConnect in the `msodbc` connection structure to [!INCLUDE [product-name](../includes/product-name.md)] using a TDS (Tabular Data Streaming) proxy service.
+data platform-specific adapter plugins allow users to connect to the data store of choice. To connect to Synapse Data Warehouse in Microsoft [!INCLUDE [product-name](../includes/product-name.md)] from dbt use `dbt-fabric` adapter. Similarly, the Azure Synapse Analytics dedicated SQL pool data source has its own adapter, `dbt-synapse`.
```
Comment (WilliamDAssafMSFT) [inline docs/data-warehouse/connectivity.md:125]:
```suggestion
The `dbt` data platform-specific adapter plugins allow users to connect to the data store of choice. To connect to Synapse Data Warehouse in Microsoft [!INCLUDE [product-name](../includes/product-name.md)] from `dbt` use `dbt-fabric` adapter. Similarly, the Azure Synapse Analytics dedicated SQL pool data source has its own adapter, `dbt-synapse`.
```

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]

**Review by dataders — COMMENTED** [review]
@WilliamDAssafMSFT long time; no see! thanks for the edits here and hope you're doing well

**Comment by WilliamDAssafMSFT** [comment]
That was fast, thanks @dataders! One more edit: please iterate the `ms.date`, then we can sign off.

**Comment by WilliamDAssafMSFT** [comment]
#assign:WilliamDAssafMSFT

File: docs/data-warehouse/connectivity.md [hunk 3]
```diff
@@ -118,13 +118,15 @@ When establishing connectivity via JDBC, check for the following dependencies:
...
+
+Both adapters support Azure Active Directory (Azure AD) authentication and allow developers to use `az cli authentication`. However, SQL authentication is not supported for `dbt-fabric`
 
-The DBT Fabric DW adapter supports Azure Active Directory (Azure AD) authentication and allows developers to use `az cli authentication` using the dbt-synapsevnext adapter. SQL Authentication is not supported.
+The DBT Fabric DW Adapter uses the `pyodbc` library to establish connectivity with the [!INCLUDE [fabric-dw](includes/fabric-dw.md)]. The `pyodbc` library is an ODBC implementation in Python language that uses [Python Database API Specification v2.0](https://peps.python.org/pep-0249/).  The `pyodbc` library directly passes connection string to the database driver through SQLDriverConnect in the `msodbc` connection structure to [!INCLUDE [product-name](../includes/product-name.md)] using a TDS (Tabular Data Streaming) proxy service.
```
Comment (WilliamDAssafMSFT) [inline docs/data-warehouse/connectivity.md:129]:
```suggestion
The DBT Fabric DW Adapter uses the `pyodbc` library to establish connectivity with the [!INCLUDE [fabric-dw](includes/fabric-dw.md)]. The `pyodbc` library is an ODBC implementation in Python language that uses [Python Database API Specification v2.0](https://peps.python.org/pep-0249/).  The `pyodbc` library directly passes connection string to the database driver through SQLDriverConnect in the `msodbc` connection structure to [!INCLUDE [product-name](../includes/product-name.md)] using a TDS (Tabular Data Streaming) proxy service. For more information, see the [Microsoft Fabric Synapse Data Warehouse dbt adapter setup](https://docs-getdbt-com.netlify.app/reference/warehouse-setups/fabric-setup) and [Microsoft Fabric Synapse Data Warehouse dbt adapter configuration](https://docs-getdbt-com.netlify.app/reference/resource-configs/fabric-configs).
```

@dataders Added a line from Predeep and Charles. Please confirm when the used links will work.

**Comment by ShannonLeavitt** [comment]
Closing/opening per email request from @WilliamDAssafMSFT

**Comment by WilliamDAssafMSFT** [comment]
@dataders Please review https://github.com/MicrosoftDocs/fabric-docs/pull/25#pullrequestreview-1456340251 and let me know when those links will work. Thanks!

File: docs/data-warehouse/connectivity.md (hunk 3 above)
Comment (dataders) [reply docs/data-warehouse/connectivity.md:129]: @WilliamDAssafMSFT so https://github.com/dbt-labs/docs.getdbt.com/pull/3461/ got merged and deployed 30 minutes ago. We're still waiting for the changes to get propogated throughout the Netfly CDN, but the updated URLs provided in my suggestion below should work within the hour. Thanks!

```suggestion
The DBT Fabric DW Adapter uses the `pyodbc` library to establish connectivity with the [!INCLUDE [fabric-dw](includes/fabric-dw.md)]. The `pyodbc` library is an ODBC implementation in Python language that uses [Python Database API Specification v2.0](https://peps.python.org/pep-0249/).  The `pyodbc` library directly passes connection string to the database driver through SQLDriverConnect in the `msodbc` connection structure to [!INCLUDE [product-name](../includes/product-name.md)] using a TDS (Tabular Data Streaming) proxy service.

For more information, see the [Microsoft Fabric Synapse Data Warehouse dbt adapter setup](https://docs.getdbt.com/docs/core/connect-data-platform/fabric-setup) and [Microsoft Fabric Synapse Data Warehouse dbt adapter configuration](https://docs.getdbt.com/docs/core/connect-data-platform/fabric-configs).
```

Comment (WilliamDAssafMSFT) [reply docs/data-warehouse/connectivity.md:129]: Great, thanks @dataders! Will sign off when I see them live.

Comment (WilliamDAssafMSFT) [reply docs/data-warehouse/connectivity.md:129]: @dataders I see the first link but not the second, can you confirm the URL?

**Comment by WilliamDAssafMSFT** [comment]
@dataders Ah. I don't have permission to commit that change in your branch, it looks like you're updating the second URL, please go ahead and commit that change, then I'll sign off.

**Comment by WilliamDAssafMSFT** [comment]
@dataders Following up to get this published. Please go ahead and commit the change in your comment.

**Comment by dataders** [comment]
sorry for the unnecessary back-and-forth. should be go to go now @WilliamDAssafMSFT!

**Pull request merged**